src/app/designer.controller.js

```diff
@@ -272,6 +274,28 @@ angular
         }
     };
 
+    $rootScope.addStaticResponse = function (endpoint_index) {
+
+        if (typeof $rootScope.service.endpoints[endpoint_index].extra_config['github.com/devopsfaith/krakend/proxy'] === "undefined") {
```

Comment:
It should do nothing, because there is already a static response. The method is only relevant to initialize a default static response. The model is changed directly in angular. Not the best name for the function, indeed.